packages/testing/src/TestStack.ts

```diff
@@ -0,0 +1,83 @@
+import { tmpdir } from 'node:os';
+import { join } from 'node:path';
+import { readFile } from 'node:fs/promises';
+import { App, Stack } from 'aws-cdk-lib';
+import { AwsCdkCli, RequireApproval } from '@aws-cdk/cli-lib-alpha';
+import type { ICloudAssemblyDirectoryProducer } from '@aws-cdk/cli-lib-alpha';
+
+/**
+ * Test stack that can be deployed to the selected environment.
+ */
+class TestStack implements ICloudAssemblyDirectoryProducer {
+  /**
+   * Reference to the AWS CDK App object.
+   * @default new App()
+   */
+  public appRef: App;
```

Comment:
Is there any specific reason for `Ref` suffix? Can we keep it as `app` and `stack`?

Reply:
This is probably a mental model I brought from React where references to DOM nodes [are called `ref`](https://legacy.reactjs.org/docs/refs-and-the-dom.html).

In this specific case I wanted to clearly signal that these are references to the CDK `Stack` and `App` objects, rather than something internal that we create.

It's also partially an answer to your feedback in the previous PR about name overloading.

Should I change them and remove the `Ref` suffix?

Reply:
Ah makes sense. I don't have a strong opinion on that. I see that we actually create the object in the constructor (TIL about `??` coalesce 👌). Imho we don't lose any value if we remove it, especially because there is no concept or usage of `Ref` suffix within CDK domain language (I might be wrong, just never had a chance to see it).

Reply:
I have removed the suffix and re-ran the integration tests, which [are passing](https://github.com/aws-powertools/powertools-lambda-typescript/actions/runs/5692859819).